applications/members/views.py

```diff
@@ -132,13 +132,14 @@ def autoSearch(request):
 def mapSearch(request):
     location = request.GET.get('search', '')
     city = location.split(',', 1)[0]
+    state= location.split(',')[1].strip()
```

Comment:
This will throw an `IndexError` if split only generates a list of 1 length. `state` should be set to `''` if no state is passed.